mountinfo/mounted_linux.go

```diff
@@ -88,7 +88,7 @@ func mountedFast(path string) (mounted, sure bool, err error) {
 
 func mounted(path string) (bool, error) {
 	path, err := normalizePath(path)
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
Is `os.IsNotExist()` still the correct way to check this, or should we use `errors.Is(err, os.ErrNotExist)` for this? (not 100% sure, ISTR `os.IsNotExist()` also takes some other cases into account)

Reply:
It should be errors.Is(err, fs.ErrNotExist) as per documentation.
Ref: https://pkg.go.dev/os#IsNotExist. I modified the code for that.